test/integration/src/main/java/io/pravega/test/integration/demo/ClusterWrapper.java

```diff
@@ -65,52 +72,58 @@
 
     private ServiceBuilder serviceBuilder;
 
-    private ScheduledExecutorService executor;
+    @Builder.Default
+    private ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
 
-    // Configuration
-    private boolean isAuthEnabled;
-    private String tokenSigningKeyBasis;
-    private boolean isRGWritesWithReadPermEnabled;
-    private int tokenTtlInSeconds;
+    @Getter
+    @Builder.Default
+    private boolean authEnabled = false;
+
+    @Getter
+    @Builder.Default
```

Comment:
`new ClustuerWrapper()` will ensure the defaults are not used. Should we make the default constructor private to enforce the builder pattern?

Reply:
Sure, I've now made the change to prevent instantiation using the default constructor.